datadog_checks_base/datadog_checks/base/checks/openmetrics/mixins.py

```diff
@@ -461,28 +484,31 @@ def process_metric(self, metric, scraper_config, metric_transformers=None):
         try:
             self.submit_openmetric(scraper_config['metrics_mapper'][metric.name], metric, scraper_config)
         except KeyError:
-            if metric_transformers is not None:
-                if metric.name in metric_transformers:
-                    try:
-                        # Get the transformer function for this specific metric
-                        transformer = metric_transformers[metric.name]
-                        transformer(metric, scraper_config)
-                    except Exception as err:
-                        self.log.warning("Error handling metric: {} - error: {}".format(metric.name, err))
-                else:
-                    self.log.debug(
-                        "Unable to handle metric: {0} - error: "
-                        "No handler function named '{0}' defined".format(metric.name)
-                    )
-            else:
```

Comment:
Why are we changing the behavior here? 🤔

Reply:
If you defined transformers but did not get a match it would silently fail instead of falling back to wildcards